### ISSUE TITLE
Video size error

### ISSUE THREAD

**Issue by saipsa** [EliasKotlyar/Xiaomi-Dafang-Hacks#554]
In the web UI, the video size is always 1600*900, even if the value that is set is different. 

Edit: the video size in settings page.

**Comment by belch85** [comment]
Interesting, I hadn't noticed that, I think you're right.  My web UI has such a slow feed from video (still image updates every X seconds) that I don't use it for much ¯\_(ツ)_/¯.

**Comment by saipsa** [comment]
@belch85 same for me. I was talking about the settings page, not the feed quality. I have updated the main post.

**Comment by tookus123** [comment]
Here's a fix for this. I'm not sure how to get this to the right people so they can update the repository.

The file to upate is /system/sdcard/www/cgi-bin/status.cgi.

Find the video section and replace with this:

```
<select name="video_size">
<option value="-W640 -H360" $(if [ "$(cat /system/sdcard/config/rtspserver.conf | grep '^RTSPH264OPTS' | grep -o 'W640')" != "" ]; then echo selected; fi)>640x360</option>
<option value="-W960 -H540" $(if [ "$(cat /system/sdcard/config/rtspserver.conf | grep '^RTSPH264OPTS' | grep -o 'W960')" != "" ]; then echo selected; fi)>960x540</option>
<option value="-W1280 -H720" $(if [ "$(cat /system/sdcard/config/rtspserver.conf | grep '^RTSPH264OPTS' | grep -o 'W1280')" != "" ]; then echo selected; fi)>1280x720</option>
<option value="-W1600 -H900" $(if [ "$(cat /system/sdcard/config/rtspserver.conf | grep '^RTSPH264OPTS' | grep -o 'W1600')" != "" ]; then echo selected; fi)>1600x900</option>
<option value="-W1920 -H1080" $(if [ "$(cat /system/sdcard/config/rtspserver.conf | grep '^RTSPH264OPTS' | grep -o 'W1920')" != "" ]; then echo selected; fi)>1920x1080</option>
</select>
```

And here's a fix for the username and password fields so it doesn't show "/" for the username and "*" for the password when don't have username and password set.

```
<input class="input" id="videouser" name="videouser" type="text" size="12" value="$(source /system/sdcard/config/rtspserver.conf; echo $USERNAME)" />
<input class="input" id="videopassword" name="videopassword" type="password" size="12" value="$(source /system/sdcard/config/rtspserver.conf; echo $USERPASSWORD)" />
```

**Comment by jmtatsch** [comment]
@tookus123 thanks for helping. can you please make a pull request? your solution to the resolution problem probably won't work for the mjpeg streamer...

**Comment by tookus123** [comment]
@jmtatsch no problem. I appreciate you're hard work on this project. I just submitted a pull request (my first one ever so let me know if i messed up). With it, i commented on the video size settings for both RTSPH264OPTS and RTSPMJPEGOPTS.

**Comment by jmtatsch** [comment]
Fixed in #562
